private/pkg/protosource/enum.go

```diff
@@ -30,13 +31,15 @@ func newEnum(
 	namedDescriptor namedDescriptor,
 	optionExtensionDescriptor optionExtensionDescriptor,
 	allowAlias bool,
+	deprecated bool,
 	allowAliasPath []int32,
 	parent Message,
 ) *enum {
 	return &enum{
 		namedDescriptor:           namedDescriptor,
 		optionExtensionDescriptor: optionExtensionDescriptor,
 		allowAlias:                allowAlias,
+		deprecated:                deprecated,
```

Comment:
This is placed in between `allowalias` and `allowAliasPath` to keep the ordering pattern consistent with the existing ones, where all options are listed first, and then their paths or locations follow.